pkg/controller/storagecluster/quickstart_constants.go

```diff
@@ -0,0 +1,201 @@
+  name: ocs-configuration
+spec:
+  version: 4.7
+  displayName: OpenShift Container Storage Configuration
+  durationMinutes: 5
+  icon: data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+  description: Learn how to configure OpenShift Container Storage to meet your deployment
+    needs.
+  prerequisites: ["Getting Started with OpenShift Container Storage"]
+  introduction: In this tour you will learn about the various configurations available
+    to customize your OpenShift Container Storage deployment.
+  tasks:
+  - title: Expand the OCS Storage Cluster
+    description: |- 
+      When we install the OCS operator we created a storage cluster, chose
+      the cluster size, provisioned the underlying storage subsystem, deployed necessary
+      drivers, and created the storage classes to allow the OpenShift users to easily
+      provision and consume storage services that have just been deployed
+      
+      When the capacity of the cluster is about to runout we will notify you.
+      **To expand the OCS storage cluster follow these steps:**
+      
+      1. Go to installed operators page and click on **OpenShift Container Storage** 
+      2. Go to storage cluster tab
+      3. Click on the **3 dots icon** 
+      4. Click on add capacity
+      5. Use the expand cluster modal if your capacity is about to runout.
+    review:
+      instructions: |-
+        ####  To verify that you have expanded your storage cluster.
+
+        Did you expand your cluster?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have expanded the OpenShift Container Storage operator!
+      failed: Try the steps again.
+  - title: Bucket Class Configuration
+    description: |-
+      **Bucket class**
+      Bucket class policy determine the bucket's data location. Its set of policies which would apply to all buckets (OBCs) created with the specific bucket class.
```

Comment:
Typos : "The bucket class policy determines... It is a set of policies which apply to all buckets..."

Reply:
fixed